src/main/webapp/index.html

```diff
@@ -10,14 +10,135 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>          <!-- Ajax, Jquery -->
     <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>             <!-- Font awesome- used for icons-->
     <script src="script.js"></script>
+    <script src="keys.js"></script>
+
     <link rel="stylesheet" href="style.css">
+
   </head>
 
-  <body onload="javascript:getUrl()">
+  <body>
 
     <!-- This is the first section of the page that contains the navbar containing the WeTime title and other home, log in buttons, etc.. -->
-    <section class="hero is-default is-bold">
-      <div class="hero-head">
+    <section class="section">
+      <div class = "container">
+        <nav class="navbar" role="navigation" aria-label="main navigation">
+          
+          <div class="navbar-brand">
+            <a class="navbar-item" href="index.html"> WeTime <i class="fas fa-heartbeat"></i> </a>
+
+            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+              <span aria-hidden="true"></span>
+              <span aria-hidden="true"></span>
+              <span aria-hidden="true"></span>
+            </a>
+          </div>
+
+          <div id="navbarBasicExample" class="navbar-menu">
+            <div class="navbar-start">
+              <a class="navbar-item"><span class="icon"><i class="fas fa-home"></i></span><span> Home</a>
+              <a class="navbar-item"><span class="icon"><i class="fas fa-question-circle"></i></span><span> Help </span></a>
+              <a class="navbar-item"><span class="icon"><i class="fas fa-address-card"></i></span><span> About</span></a> 
+            </div>
+          </div>
+
+          <div class="navbar-end">
+            <div class="navbar-item">
+              <div class="buttons">
+                <button class = "button is-info" id="authorize_button" style="display: none;"> Login with Google</button>
+                <button class = "button is-primary" id="signout_button" style="display: none;">Log out</button>
+                <!-- <a class="button is-info"><strong>Log in</strong></a> -->
+              </div>
+            </div>
+          </div>
+        </nav>
+      </div>
+    </section>
+
+
+    <!-- <button id="authorize_button" style="display: none;">Authorize</button>
+    <button id="signout_button" style="display: none;">Sign Out</button> -->
+
+    <script type="text/javascript">
```

Comment:
I think it would be better if created a JS file for this. There's nothing technically wrong with putting JS in html but it's not good encapsulation.